woocommerce/woocommerce
# create screenshot on e2e test failure

### Changes proposed in this Pull Request:

This PR adds screenshot on test failure support to the `e2e-environment` package. Screenshots are stored in the `tests/e2e/screenshots` folder.

See #27788 .

### How to test the changes in this Pull Request:

1. Run `npm run build:packages`
2. Run all e2e tests
3. Run `WC_E2E_SCREENSHOTS=1 npx wc-e2e test:e2e tests/e2e/specs/activate-and-setup/setup-onboarding.test.js`
4. This should create the `screenshots` folder and a `.png` for the failed initial store setup test
5. Repeat step 2 and monitor the `screenshots` folder to verify that the folder is cleared of screenshots at the beginning of the test run

### Other information:


### Changelog entry

N/A
